monitoring/prometheus/alerts/test_alerts.yml

```diff
@@ -0,0 +1,774 @@
+rule_files:
```

Comment:
could you please include 8d2f39e6c568afb6880689160212bcc93057e194 in this backport as well?

Reply:
Done.